shell/platform/darwin/ios/framework/Source/platform_message_response_darwin.mm

```diff
@@ -4,6 +4,47 @@
 
 #import "flutter/shell/platform/darwin/ios/framework/Source/platform_message_response_darwin.h"
 
+/// A proxy object that behaves like NSData represented in a Mapping.
+/// This isn't a subclass of NSData because NSData is in a class cluster.
+@interface FlutterMappingData : NSObject
```

Comment:
Because NSObject gives us `forwardingTargetForSelector`, the NSProxy is slower.  Check out the documentation on `forwardingTargetForSelector` https://developer.apple.com/documentation/objectivec/nsobject/1418855-forwardingtargetforselector